app/controllers/admin/authorised_email_domains_controller.rb

```diff
@@ -26,6 +26,13 @@ def create
     end
   end
 
+  def destroy
+    authorised_email_domain = AuthorisedEmailDomain.find_by(id: params.fetch(:id))
```

Comment:
use the sanitised params: `authorised_email_params` from L38 rather than the raw `params`